styles/Editor.css

```diff
@@ -962,3 +962,7 @@
   line-height: 1.43;
   padding-left: 65px;
 }
+div[style*="background-color: rgba(249, 2, 2, 0.53)"] {
```

Comment:
Why was that added? Was it not possible to change the background-color on the element directly? @nghazzawi-atypon

Reply:
@mbartenev-atypon, please review the rollback at https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/191.